src/screens/Exchange/Exchange.js

```diff
@@ -455,8 +463,11 @@ class ExchangeScreen extends React.Component<Props, State> {
 
   setInitialSelection = (fromAssetCode: string, toAssetCode?: string, fromAmount?: number) => {
     const { assets, supportedAssets } = this.props;
+    const fromAsset = checkFiatCurrency(fromAssetCode) ?
+      fiatCurrencies.find(currency => currency.symbol === fromAssetCode) :
+      assets[fromAssetCode];
```

Comment:
Small detail: Since we search for the symbol on the `fiatCurrencies` as a way of knowing if a symbol is fiat, I'd avoid doing it twice:

```js
const fromAsset =  fiatCurrencies.find(
  currency => currency.symbol === fromAssetCode
) || assets[fromAssetCode];
```

Reply:
Updated in last commit 👍